infrastructure/output.tf

```diff
@@ -13,3 +13,27 @@ output "idam_url" {
 output "claim_store_url" {
   value = "${local.claimStoreUrl}"
 }
+
+output "feature_admissions" {
```

Comment:
another way that's more full proof i.e. doesn't rely on people adding new ones here is to hit the /info endpoint, it has all feature toggles there

Reply:
oh yes!!! yep - much better 👍